wire-ios/Wire-iOS/Sources/UserInterface/E2EIdentity/SuccessfulCertificateEnrollmentViewController.swift

```diff
@@ -128,7 +131,9 @@ final class SuccessfulCertificateEnrollmentViewController: UIViewController {
             view.addSubview($0)
         }
 
-        [detailsLabel,
+        [titleLabel,
+         shieldImageView,
+         detailsLabel,
```

Comment:
We should remove `titleLabel` and `shieldImageView` from the main view (line #125 and #126)